src/dotgraph.cpp

```diff
@@ -289,12 +287,10 @@ void DotGraph::writeGraphHeader(TextStream &t,const QCString &title)
   {
     t << "  bgcolor=\"transparent\";\n";
   }
-  t << "  edge [fontname=\"" << fontName << "\","
-         "fontsize=\"" << fontSize << "\","
-         "labelfontname=\"" << fontName << "\","
-         "labelfontsize=\"" << fontSize << "\"];\n";
-  t << "  node [fontname=\"" << fontName << "\","
-         "fontsize=\"" << fontSize << "\",shape=record];\n";
+  QCString c = Config_getString(DOT_COMMON_ATTR) + " ";
+  t << "  graph [" << Config_getString(DOT_GRAPH_ATTR) << "];\n";
+  t << "  edge [" << c << Config_getString(DOT_EDGE_ATTR) << "];\n";
+  t << "  node [" << c << Config_getString(DOT_NODE_ATTR) << "];\n";
```

Comment:
Where is `c` used?
In the original for edges the `fontname` and `labelfontname` are set by means of the `fontName`, but in the compattibility code only `fontname` is set  (analogous for `fontsize`)

Reply:
I don't see a solution or explanation here yet.
In general don set conversations to resolved, I think this should be done by the person who wrote the request when he accepts the answer.